source/reference/verification/embedded.txt

```diff
@@ -0,0 +1,101 @@
+
+.. _c2c-embedded-verifier:
+
+=============================
+Verify with Embedded Verifier
```

Comment:
I think we need to be really clear at the top here about which GA features it doesn't verify. I know it says it below, but I think we should put it in a colored box

Reply:
Fixed